src/Hardware/Battery.cpp

```diff
@@ -43,11 +43,13 @@ GetInfo() noexcept
   Info info;
   auto &battery = info.battery;
   auto &external = info.external;
-  KoboModel kobo_model = DetectKoboModel();
 
   char line[256];
 
-  if (kobo_model == KoboModel::GLO_HD) {
+  switch (DetectKoboModel())
+  {
+  case KoboModel::GLO_HD:
+  case KoboModel::CLARA_HD:
```

Comment:
Don't do code refactoring in the middle of a patch that is supposed to do something else. This causes a confusing commit diff which is hard to review. You can refactor code, but do it in a separate commit. In this diff, I can't see what really belongs to the backlight feature.

Reply:
Yes, it grew into confusion.  I have split the PR into 3 well defined commits (the first being refactoring).